scripts/qa_crowbarsetup.sh

```diff
@@ -1135,9 +1210,12 @@ function onadmin_prepare_cloud_repos()
                 addsp3testupdates
                 addsles12testupdates
                 ;;
-            develcloud6|develcloud7|mitakacloud7|susecloud7|M?|Beta*|RC*|GMC*)
+            develcloud6)
                 addsles12sp1testupdates
                 ;;
+            develcloud7|mitakacloud7|susecloud7|M?|Beta*|RC*|GMC*)
+                addsles12sp2testupdates
+                ;;
```

Comment:
we dont have testupdates atm, so this is mostly a NOP. except if you wanted to test experimental SP1 updates, but we already cover those with cloud6